Measuring-success/Using-data-to-improve-your-service.markdown

```diff
@@ -35,117 +35,9 @@ Think about:
 
 ## Web measurement framework 
 
-This is an example of a web measurement framework we use at Essex County Council (ECC) for measuring performance.  
-
-It forms the basis of reports and dashboards and will change slightly from service to service. 
-
-### Objective 1: provide engaging content  
-
-<div class="table">
-<table>
-  <caption>The goal is to increase content engagement.</caption>
-    <thead>
-      <tr>
-        <th scope="col">KPIs</th>
-        <th scope="col">KPI target</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <td>Scroll depth</td>
-        <td>50%</td>
-      </tr>
-      <tr>
-        <td>Average on time page</td>
-        <td>2.54 minutes</td>
-      </tr>
-      <tr>
-        <td>Number of feedback comments</td>
-        <td>2</td>
-      </tr>
-      <tr>
-        <td>Pages per session</td>
-        <td>3</td>
-      </tr>
-    </tbody>
-</table>
-</div>
-
-### Objective 2: provide effective content
-
-<div class="table">
-<table>
-  <caption>The goal is to increase content effectiveness.</caption>
-    <thead>
-      <tr>
-        <th scope="col">KPIs</th>
-        <th scope="col">KPI target</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <td>Number of clicked on external links</td>
-        <td>205</td>
-      </tr>
-      <tr>
-        <td>Number of document downloads </td>
-        <td>110</td>
-      </tr>
-      <tr>
-        <td>Number of widget completions</td>
-        <td>10</td>
-      </tr>
-      <tr>
-        <td>Number of completed assessment forms</td>
-        <td>7</td>
-      </tr>
-      <tr>
-        <td>Number of video views</td>
-        <td>36</td>
-      </tr>
-      <tr>
-        <td>Bounce rate</td>
-        <td>40%</td>
-      </tr>
-      <tr>
-        <td>Number of internal search</td>
-        <td>120</td>
-      </tr>
-    </tbody>
-</table>
-</div>
-
-### Objective 3: knowledge base
-
-<div class="table">
-<table>
-  <caption>The goal is to increase website traffic.</caption>
-    <thead>
-      <tr>
-        <th scope="col">KPIs</th>
-        <th scope="col">KPI target</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <td>Total number of monthly visits</td>
-        <td>15,000</td>
-      </tr>
-      <tr>
-        <td>Number of visits for each acquisition channel</td>
-        <td>Segmented</td>
-      </tr>
-      <tr>
-        <td>Total number of visits carrying out a goal or event</td>
-        <td>Segmented</td>
-      </tr>
-      <tr>
-        <td>Number of returning visits compared to new visits </td>
-        <td>Segmented</td>
-      </tr>
-    </tbody>
-</table>
-</div>
+A web measurement forms the basis of reports and dashboards and will change slightly from service to service. 
+
+[See an example of a web measurement framework we use at Essex County Council (ECC) for measuring performance](https://keelanfh.github.io/essex-county-council-digital-manual/Measuring-success/Google-analytics/Ecc-web-analytics-framework).
```

Comment:
This looks to be the only instance of it but worth checking to confirm the links are relative or direct to the digital manual

Reply:
Oops, great spot, thanks! Should be fixed now. 🤭